ci/testgrid/config.yaml

```diff
@@ -255,6 +271,18 @@ dashboards:
   - name: istio-1.1.7-no-mesh
     test_group_name: ci-knative-serving-istio-1.1.7-no-mesh
     base_options: "sort-by-name="
+  - name: k8s-1.12-istio-1.1
```

Comment:
I think we should start thinking about grouping all these variations in a separate tab/group...